feat(dynamic-sampling): add link from accepted span card to dynamic sampling settings if applicable

constantinius opened the pull request:
In Stats add a link from the "accepted spans" card to the dynamic sampling settings if dynamic sampling was applied


Closes https://github.com/getsentry/projects/issues/140

constantinius committed:
feat(dynamic-sampling): add link from accepted span card to dynamic sampling settings if applicable

matejminar commented on static/app/views/organizationStats/usageStatsOrg.tsx:
```diff
@@ -608,3 +610,35 @@ const FooterDate = styled('div')`
+
+function SpansStored({organization, acceptedStored}: SpansStoredProps) {
+  return (
+    <div style={{gap: '8px', display: 'inline-grid', gridAutoFlow: 'column'}}>
```
We should avoid inline styles otherwise things go out of hand quite quickly.
You can completely remove this if you append `{' '} ` to line 630.

constantinius replied:
Done with [00bd043](https://github.com/getsentry/sentry/pull/81311/commits/00bd043c23d278f8376da3d5c7fa11003ebb2ae7)

matejminar commented on static/app/views/organizationStats/usageStatsOrg.tsx:
```diff
@@ -608,3 +610,35 @@ const FooterDate = styled('div')`
+
+const StyledSettingsButton = styled(LinkButton)`
+  margin-top: -${space(0.5)};
+  padding: 3px;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
```
Let's not override the style of our default buttons if possible.

constantinius replied:
Done with [00bd043](https://github.com/getsentry/sentry/pull/81311/commits/00bd043c23d278f8376da3d5c7fa11003ebb2ae7)

matejminar reviewed (commented):
![image](https://github.com/user-attachments/assets/5aa943bc-fee4-4207-aeea-71f26871d646)

The hover state seems to be cut off.

Left a few comments, other than that, lgtm!

constantinius committed:
fix(dynamic-sampling): remove unnecessary styling

Use default styling of button, removed unnecessary inline styles, fixed layout of SpansStored view

constantinius committed:
Merge branch 'master' into constantinius/feat/dynamic-sampling/accepted-spans-add-link-to-ds-settings

constantinius committed:
fix(dynamic-sampling): small styling fixes
